File: JS-Basics/04-Conditional-Statements-Exercise/p08_Lunch_Break.js
function lunchBreak(input) {
    let nameSerial = input[0];
    let lenghtEpizod = Number(input[1]);
    let lenghtBreak = Number(input[2]);

    let lunchTime = lenghtBreak / 8;
    let breakTime = lenghtBreak / 4;

    let freeTime = lenghtBreak - (lunchTime + breakTime);

    if (freeTime >= lenghtEpizod) {
        console.log(`You have enough time to watch ${nameSerial} and left with ${Math.ceil(freeTime - lenghtEpizod)} minutes free time.`);
    } else {
        console.log(`You don't have enough time to watch ${nameSerial}, you need ${Math.ceil(lenghtEpizod - freeTime)} more minutes.`);
    }
}

lunchBreak(["Game of Thrones", "60", "96"]);
lunchBreak(["Teen Wolf", "48", "60"]);

File: Programming-Basic/04-Conditional-Statements-Exercise/p08_Lunch_Break.java
import java.util.Scanner;

public class p08_Lunch_Break {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String nameSerial = scanner.nextLine();
        int timeLine = Integer.parseInt(scanner.nextLine());
        int timeBreak = Integer.parseInt(scanner.nextLine());

        double lunchTime = timeBreak * 0.125;
        double breathTime = timeBreak * 0.25;
        double restTime = timeBreak - lunchTime - breathTime;

        if (restTime >= timeLine) {
            System.out.printf("You have enough time to watch %s and left with %.0f minutes free time.", nameSerial, Math.ceil(restTime - timeLine));
        } else {
            System.out.printf("You don't have enough time to watch %s, you need %.0f more minutes.", nameSerial, Math.ceil(timeLine - restTime));
        }

    }
}
